internal/website/content/howto/secrets/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Secrets"
+date: 2019-03-21T17:42:18-07:00
+showInSidenav: true
+draft: true
+---
+
+Cloud applications frequently need to store sensitive information like web
```

Comment:
Mind spending an extra paragraph or two here on the motivation and the setup? With my cloud newbie hat on, it took a bit to understand what these services actually do (after all, Go already has good crypto in the stdlib)

Reply:
Done.